copier/main.py

```diff
@@ -643,6 +644,7 @@ def run_update(self) -> None:
                 f"Your are downgrading from {self.subproject.template.version} to {self.template.version}. "
                 "Downgrades are not supported."
             )
+        print(f"Updating from template version {self.template.commit}")
```

Comment:
Same as above, plus removing that "from" which I think is a bit misleading:

```suggestion
        print(f"Updating to template version {self.template.version}", file=sys.stderr)
```